documentation/.yarnrc.yml

```
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

```suggestion
nodeLinker: node-modules

yarnPath: .yarn/releases/yarn-3.2.0.cjs
```

We wanted to use the same version everywhere